source/test/fixtures/module-resolution/node16-from-package-json/package.json

```diff
@@ -0,0 +1,9 @@
+{
+	"name": "foo",
```

Comment:
I don't think it matters, but for correctness, this and the others, should have `"type": "module"`.